ops/productionize

```diff
@@ -7,6 +7,11 @@ set -o pipefail
 #
 SRC="$(git rev-parse --show-toplevel)"
 
+# Confirm that the binary is present and working.
+#
+BIN="$SRC"/rust/target/release/mina-indexer
+"$BIN" --help
```

Comment:
Printing the `--version` would be more useful methinks

Reply:
Okay.